### DISCUSSION TITLE
❌ Allow hidden endpoints

### DISCUSSION THREAD

**Discussion by eluce2** [RobinTail/express-zod-api#1560] [Ideas]
I would like to create an endpoint in my API for internal use only. It would be create if I could tag that endpoint with something like "Internal" then hide it from the OpenAPI spec and my generated documentation page

**Comment by RobinTail** [comment]
Hello @eluce2 , thanks for the idea. Btw and FYI,

The `routing` property of the `Documentation::constructor()` argument gives you control on the endpoints to include into the generated documentation in case you differentiate some of its parts as private or undisclosed.
Meanwhile, the `routing` supplied to `createServer()` can extend the one mentioned above. 

```ts
const publicRoutes: Routing = {}; // used by Documentation
const privateRoutes: Routing = {};
const allRoutes = { ...publicRoutes, ...privateRoutes }; // used by createServer
```
